result/arrow-parens.js

```diff
@@ -0,0 +1,94 @@
+/*eslint-env es6*/
+
+// Bad
+a => {}
+
+// Good
+(a) => {}
+/*eslint-env es6*/
+
+// Bad
+if (a => 2) {
```

Comment:
__[development][eslint]__
* Error - Unexpected constant condition. (no-constant-condition)
* Error - 'a' is defined but never used (no-unused-vars)
* Error - Empty block statement. (no-empty)